packages/holocron-module-register-webpack-plugin/HolocronModuleRegisterPlugin.js

```diff
@@ -15,14 +15,15 @@
 const { ConcatSource } = require('webpack-sources');
 const ModuleFilenameHelpers = require('webpack/lib/ModuleFilenameHelpers');
 
-function HolocronModuleRegisterPlugin(moduleName) {
+function HolocronModuleRegisterPlugin(moduleName, holocronModuleName = 'holocronModule') {
   this.moduleName = moduleName;
+  this.holocronModuleName = holocronModuleName;
```

Comment:
What is the difference between this new `holocronModuleName` and the existing `moduleName` var?

Reply:
holocronModule name is hypenated moduleName to prevent web pack chunk clashing.